app/Commands/ConvertToAcceptance.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Commands;
+
+use App\Conference;
+use App\Talk;
+
+class ConvertToAcceptance extends Command
```

Comment:
Disclaimer: I haven't worked with Symposium in a while, so I may just be mis-remembering the architecture.

My gut on this said I'd rename it to something like "ConvertSubmissionToAcceptance."

But when I started typing that, I started wondering, why are we converting at all? It's not not-submitted once it's accepted. So what makes more sense to my brain is one of the following:

1. Add metadata to a submission. Maybe an `is_accepted` column.
OR
2. Don't convert a submission to an acceptance, but rather *add* an acceptance, so that a submitted talk is always submitted, and sometime's it's *also* accepted.